src/Autofac.Extras.Moq/Autofac.Extras.Moq.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <Description>Autofac extension for automocking and creation of mock objects in Moq.</Description>
-    <VersionPrefix>5.0.1</VersionPrefix>
+    <VersionPrefix>6.0.0</VersionPrefix>
```

Comment:
Should the build be updated with the .csproj patch thing we have in core Autofac so this isn't required anymore?